src/guides/nft-marketplace/index.md

```diff
@@ -589,7 +589,6 @@ After changing the account number, you can execute the script again to test: `tr
 If you don't want to use Dashboards, you can modify your Truffle config files to use environment variables you set up. Create a `.env` file with the following code:
```

Comment:
Not sure if these updates should also be in this PR? 🤔

Reply:
The optimism bridge tutorial references the original marketplace tutorial, so that's why I did a double update

Reply:
👍